setup.py

```diff
@@ -0,0 +1,12 @@
+from distutils.core import setup
+
+setup(
+    name='nerdouts',
+    version='0.1dev',
+    packages=['nerdouts'],
+    license='Creative Commons Attribution-Noncommercial-Share Alike license',
```

Comment:
TIL about this license..

Reply:
Same. This was definitely carefully chosen and not copy/pasted...